README.md

```diff
@@ -0,0 +1,13 @@
+# jsonnext
+
+Extension libraries for [go-jsonnet](https://github.com/google/go-jsonnet).
+The name `jsonnext` is a contraction of `jsonnet` and `ext`.
+
+Godoc documentation is at https://pkg.go.dev/foxygo.at/jsonnext.
```

Comment:
Make it a link too?
`[pkg.go.dev/foxygo.at/jsonnext](https://pkg.go.dev/foxygo.at/jsonnext)`

Reply:
I figured any markdown renderer would make this a link anyway, but perhaps best to be explicit. Not sure about using `pkg.go.dev/foxygo.at/jsonnext` as the link text - it looks too much like an import path without the scheme. For something so simple, I'm having trouble coming up with link text that i like.